libs/langchain/langchain/chat_loaders/imessage.py

```diff
@@ -107,8 +107,13 @@ def lazy_load(self) -> Iterator[chat_loaders.ChatSession]:
             ) from e
         cursor = conn.cursor()
```

Comment:
Might be good to add a context manager for the cursor to close the connection on error.